skins/moono-lexicon/icons/icons.json

```diff
@@ -4,36 +4,59 @@
         "align-image-center": "align-image-center",
         "align-image-left": "align-image-left",
         "align-image-right": "align-image-right",
-        "anchor": "flag-full",
+        "anchor": {
+            "ltr": "flag-full"
+        },
         "blockquote": "quote-right",
```

Comment:
Shouldn't this one be directional as well? :D

Reply:
It is but we don't have the rtl icon yet. I can use the same until we have it or just leave it blank.